packages/node_modules/@node-red/nodes/locales/en-US/function/10-switch.html

```diff
@@ -38,8 +38,8 @@ <h4>Rules</h4>
     <h4>Notes</h4>
     <p>The <code>is true/false</code> and <code>is null</code> rules perform strict
        comparisons against those types. They do not convert between types.</p>
-    <p>The <code>is empty</code> rule passes for Strings, Arrays and Buffers that have
-       a length of 0, or Objects that have no properties. It does not pass for <code>null</code>
+    <p>The <code>is empty</code> and <code>is not empty</code> rules passes for Strings, Arrays and Buffers that have
+       a length of 0, or Objects that have no properties. It does not pass for <code>boolean</code>, <code>null</code> 
        or <code>undefined</code> values.</p>
```

Comment:
The proposed update isn't quite true as it says `is not empty` will pass for 0-length Strings etc. Can I suggest this alternative?

```suggestion
    <p>The <code>is empty</code> and <code>is not empty</code> rules can be used to test the length of Strings, Arrays and Buffers, or the number of properties an Object has. Neither rule will pass if the property being tested has a <code>boolean</code>, <code>null</code> 
       or <code>undefined</code> value.</p>
```

@kazuhitoyokoi FYI - this will require translating once it has merged.

Reply:
Looks good to me ! (Sorry for the delay was on vacation)